unittests/v1.2.0/test_report_runner.py

```diff
@@ -0,0 +1,73 @@
+from compliance_suite.report_runner import *
+import re
+
+expect_final_json = json.loads(
+    open("unittests/output/expected_good.json", "r").read()
+)
+
+def string_format(json):
+    return str(json).replace("'", '"').replace("\\","")
+
+expect_json_s = string_format(expect_final_json)
+
+def remove_timestamps(actual_final_json):
+    # remove timestamps, otherwise assert will fail 100%
+    actual_final_json["start_time"] = ""
+    actual_final_json["end_time"] = ""
+    for phase in actual_final_json["phases"]:
+        phase["start_time"] = ""
+        phase["end_time"] = ""
+        for test in phase["tests"]:
+            test["start_time"] = ""
+            test["end_time"] = ""
+            for case in test["cases"]:
+                case["start_time"] = ""
+                case["end_time"] = ""
+    return string_format(actual_final_json)
+
+def validate_timestamp_format(timestamp):
+    good_timestamp = re.search("(\d{4}-\d{2}-\d{2})T(\d{2}:\d{2}:\d{2})Z", timestamp)
+    return good_timestamp != None
+
+#def test_report_runner():
```

Comment:
What is this commented out function? Is it meant to be removed?

Reply:
test_report_runner() was the previous line by line, end-to-end testing so if we decide to include it instead of going by # passing tests we could